### DISCUSSION TITLE
[Question] does LLaVA-Lightning-MPT-7B-preview lack of some file?

### DISCUSSION THREAD

**Discussion by bofei5675** [haotian-liu/LLaVA#150] [Q&A]
https://huggingface.co/liuhaotian/LLaVA-Lightning-MPT-7B-preview/tree/main
After downloading all files from above link, I try to run the inference scripts, and find that tokenizer.model is not in that.

Anyone know what's going on here?

**Comment by haotian-liu** [comment]
Hi @bofei5675, please paste the commands, and error logs here.  And you can run the command without downloading for this MPT model specifically.

**Comment by lucas-0liveira** [comment]
It seems like the issue you're facing is related to the absence of the "tokenizer.model" file after downloading the files from the provided link. It's possible that there might be an error in the project's structure, or the file may have been removed or relocated since the last update.
I recommend checking if there's any documentation or additional instructions in the repository that might shed light on this issue.
